src/sentry/snuba/discover.py

```diff
@@ -494,6 +494,8 @@ def get_row(row):
     "latest_event": "latest_event()",
     "apdex": "apdex(300)",
     "impact": "impact(300)",
+    "rpm": "epm()",
+    "rps": "eps()",
```

Comment:
Do any users have rpm/rps in their saved queries?

Reply:
Nobody has it in their queries right now.